Samples~/E2ELatency/BarcodeDecoder.cs

```diff
@@ -0,0 +1,49 @@
+using UnityEngine;
+
+public class BarcodeDecoder : MonoBehaviour
+{
+    [SerializeField] int Row;
+    [SerializeField] int Column;
+    [SerializeField] ComputeShader Shader;
+
+    GraphicsBuffer readbackBuffer_;
+    Color[] data_;
+
+    private void Awake()
+    {
+        int count = Row * Column;
+        int stride = sizeof(float) * 4;
+        readbackBuffer_ =
+            new GraphicsBuffer(GraphicsBuffer.Target.Structured, count, stride);
+        data_ = new Color[count];
+    }
+
+    private void OnDestroy()
+    {
+        readbackBuffer_.Dispose();
+    }
+
+    public int GetValue(Texture texture)
+    {
+        return Decode(texture);
+    }
+
+    int Decode(Texture source)
+    {
+        Shader.SetTexture(0, "Source", source);
+        Shader.SetInt("Row", Row);
+        Shader.SetInt("Column", Column);
+        Shader.SetBuffer(0, "Result", readbackBuffer_);
+        Shader.Dispatch(0, Row, Column, 1);
```

Comment:
Isn't it necessary to use the result of `Shader.FindKernelIndex`?

Reply:
fixed it